### ISSUE TITLE
Control point editor causes the program to crash when deleting points at the end of a line

### ISSUE THREAD

**Issue by CiprianB24** [opentoonz/opentoonz#4106]
When you try to delete the points at the ends of a line the, program crashes
I have been working with OpenToonz for about a year and this issue is the most annoying thing at this program. I often use this tool because it gives me the most control in modeling the shape of every vector drawing.

Reproducing the bug:
1. draw a random line on the sheet
2. select the "Control Point Editor Tool" from the tool bar (or press "C")
3. select the line
3. hold down the "Delete" button and click the dots at the end of the line in order to wipe them
4. repeat this process until the program randomly crashes 

 - Windows 10
 - OpenToonz Version: Windows  v1.5
 - GPU: Intel(R) HD Graphics 520
 - GPU Driver Version: 27.20.100.8681

**Comment by shd8698766** [comment]
Yes it crashes, if you drag the cursor while pressing the delete button the program won't crash.

**Comment by CiprianB24** [comment]
> Yes it crashes, if you drag the cursor while pressing the delete button the program won't crash.

I did as you told me and OpenToonz crashed. it looks like the whole problem is about dragging points when deleting them

**Comment by shd8698766** [comment]
yes, anyway your issue is real as I can replicate it.

- Windows10
- OpenToonz 1.5
